web/themes/contrib/civictheme/assets/sass/theme.admin.scss

```diff
@@ -0,0 +1,6 @@
+//
+// Styles used for when the CivicTheme is rendered in the admin theme.
+//
+// These styles will not be visible in the Storybook.
+@import 'layout_builder/layout_builder';
```

Comment:
the administration style are now built as a separate asset and included as a library for authenticated user

Reply:
This is nice separating out Drupal specific CSS from uikit